wrappingGsdk/README.md

```diff
@@ -0,0 +1,56 @@
+# Wrapping an existing game using GSDK
+
+There are cases in which you want to wrap an existing game with a custom process that uses Multiplayer Servers Game Server SDK (GSDK). Specifically, this process will spawn your game server executable and integrate with GSDK. You may want to create this wrapper for one of the following reasons:
+
+- you already have an existing game (or building a new one) and you want to try Multiplayer Servers service with the minimum possible effort
+- you want to evaluate the MPS platform (in this case you may also see our [OpenArena sample](./OpenArena/README.md))
+
+> This sample and corresponding technique is NOT meant to be used in production but only for evaluation/development purposes. Proper integration of your game server with the GSDK is highly recommended.
+
+## Wrapping an existing game server using the wrapper app
+
+To get started, you can find two .NET Core projects in the current folder. 
+- `wrapper` is a .NET Core console application that acts as a wrapper for your game server and integrates with GSDK using the [latest Nuget package](https://www.nuget.org/packages/com.playfab.csharpgsdk)
+- `fakegame` is a .NET Core console application that keeps itself alive for ever. It's meant to simulate a game server that has absolutely zero knowledge of GSDK. You can use it if you don't have a game server of your own.
+
+### Building the wrapper
```

Comment:
consider a powershell script that does all this (builds both apps and creates a zip file).